repos/fdbt-site/src/pages/products/productDetails.tsx

```diff
@@ -301,7 +303,11 @@ const createProductDetails = async (
     if ('lineId' in ticket) {
         productDetailsElements.push({
             name: 'Fare triangle',
-            content: ['You created a fare triangle'],
+            content: [
+                fareTriangleModified
+                    ? `Updated: ${convertDateFormat(fareTriangleModified?.toString())}`
```

Comment:
shouldnt need the ? on the end of fareTriangleModified, as we know its not undefined here.